src/components/media-controls/readme.md

```diff
@@ -7,11 +7,15 @@
 
 ## Properties
 
-| Property | Attribute | Description     | Type     | Default     |
-| -------- | --------- | --------------- | -------- | ----------- |
-| `first`  | `first`   | The first name  | `string` | `undefined` |
-| `last`   | `last`    | The last name   | `string` | `undefined` |
-| `middle` | `middle`  | The middle name | `string` | `undefined` |
+| Property    | Attribute | Description | Type           | Default     |
+| ----------- | --------- | ----------- | -------------- | ----------- |
+| `back`      | `back`    |             | `string`       | `'Back'`    |
+| `for`       | `for`     |             | `string`       | `undefined` |
+| `forward`   | `forward` |             | `string`       | `'Forward'` |
+| `onBack`    | --        |             | `VoidFunction` | `undefined` |
+| `onForward` | --        |             | `VoidFunction` | `undefined` |
```

Comment:
Fairly certain these were removed, update the properties. Potentially update git hooks to automatically update these files when you commit